src/components/todos/Filter/Filter.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { Status } from '../../../types/Status';
+import { Todo } from '../../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  selected: string;
+  setStatus: (status: Status) => void;
+  removeCompleted: (completedTodos: Todo[]) => void;
+}
+
+export const Filter: React.FC<Props> = ({
+  todos,
+  selected,
+  setStatus,
+  removeCompleted,
+}) => {
+  const statuses = [Status.All, Status.Active, Status.Completed];
+
+  const getCompletedTodos = () => todos.filter(todo => (
+    todo.completed
+  ));
+
+  const completedTodos = useMemo(getCompletedTodos, [todos, selected]);
+
+  const filterCompleted = () => (
+    removeCompleted(completedTodos)
+  );
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {todos.filter(todo => !todo.completed).length}
+        {' items left'}
```

Comment:
Better create a separate variable for `todos.filter(todo => !todo.completed).length` and use [interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here.